client/my-sites/hosting/web-server-logs-card/index.js

```diff
@@ -60,54 +253,58 @@ const WebServerLogsCard = ( props ) => {
 		},
 	];
 
-	useEffect( () => {
-		const startMoment = moment.utc( startDateTime );
-		const endMoment = moment.utc( endDateTime );
-
-		const startDateIsValid =
-			startMoment.isValid() && startDateTime === startMoment.format( dateTimeFormat );
-		const endDateIsValid =
-			endMoment.isValid() && endDateTime === endMoment.format( dateTimeFormat );
-
-		setStartDateValidation( {
-			isValid: startDateIsValid,
-			validationInfo: startDateIsValid
-				? translate( 'Date is valid' )
-				: translate( 'Start date format is not valid.' ),
-		} );
-
-		setEndDateValidation( {
-			isValid: endDateIsValid,
-			validationInfo: endDateIsValid
-				? translate( 'Date is valid' )
-				: translate( 'End date format is not valid.' ),
-		} );
-
-		if ( ! startDateIsValid || ! endDateIsValid ) {
-			return;
-		}
-
-		if ( ! startMoment.isBefore( endMoment ) ) {
-			setStartDateValidation( {
-				isValid: false,
-				validationInfo: translate( 'Start date must be earlier than end date.' ),
-			} );
-		}
-
-		if ( startMoment.isBefore( moment.utc().subtract( 14, 'days' ) ) ) {
-			setStartDateValidation( {
-				isValid: false,
-				validationInfo: translate( 'Start date must be less than 14 days ago.' ),
-			} );
-		}
-	}, [ startDateTime, endDateTime ] );
-
-	const updateStartDateTime = ( event ) => {
-		setStartDateTime( event.target.value );
-	};
+	const onValidateInputs = useCallback(
+		( start, end ) => {
+			const startMoment = moment( start, localeDateFormat );
+			const endMoment = moment( end, localeDateFormat );
+			const startDateIsValid = startMoment.isValid();
+			const endDateIsValid = endMoment.isValid();
+
+			let startValidatorResults = {
+				isValid: startDateIsValid,
+				validationInfo: startDateIsValid
+					? translate( 'Date is valid' )
```

Comment:
I don't think we need to display a message when the date is valid. It's a little bit redundant.

Reply:
I totally agree. And this was my initial approach.
However the `FormInputValidation` component, uses a checkmark in case of a non-error state, and it looks a bit ugly if there is no text there.
We can take another approach though, and hide the component in case we don't have an error.
If you think this is better, I can do that.

Reply:
> uses a checkmark in case of a non-error state, and it looks a bit ugly if there is no text there.
> We can take another approach though, and hide the component in case we don't have an error.
> If you think this is better, I can do that.

@kozer Ah, ok. I don't have a strong opinion about it and am fine with either option.